docs/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# An docker image `databricks/sparkdl` need to be generated by run
+#     $docker build -t databricks/sparkdl --build-arg PYTHON_VERSION=2.7 .
+# on base directory. Currently we need to build it with Python2.
+
+FROM databricks/graphframes
+
+RUN sudo apt-get install -y python-pip && \
```

Comment:
* `sudo` is not needed
* Always do

~~~
apt-get updates &&
...
apt-get clean
~~~

Because docker will cache the layer and the apt cache might not be not up-to-date.